dev/benchmarks/macrobenchmarks/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -277,7 +265,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "echo \"$PRODUCT_NAME.app\" > \"$PROJECT_DIR\"/Flutter/ephemeral/.app_filename\n";
+			shellScript = "echo \"$PRODUCT_NAME.app\" > \"$PROJECT_DIR\"/Flutter/ephemeral/.app_filename\n\"$FLUTTER_ROOT\"/packages/flutter_tools/bin/macos_assemble.sh embed\n";
```

Comment:
We should `set -e` or use `&&`; it's very unlikely the echo would fail, but there could potentially be permission issues with the piping.